apps/omg_watcher_rpc/lib/web/views/transaction.ex

```diff
@@ -61,7 +61,9 @@ defmodule OMG.WatcherRPC.Web.View.Transaction do
     outputs = Map.fetch!(transaction, :outputs)
 
     transaction
-    |> Map.take([:txindex, :block, :txhash, :metadata])
+    |> Map.take([:txindex, :txhash, :block, :inputs, :outputs, :txbytes, :metadata])
+    |> Map.update!(:inputs, &render_txoutputs/1)
+    |> Map.update!(:outputs, &render_txoutputs/1)
     |> Map.put(:results, digest_outputs(outputs))
```

Comment:
just backwards-compatability

Reply:
Backwards is a necessary evil and often ends up being a pain. And deprecation timelines can go on for ages. Given this will be an initial release I don't see the value in maintaining any backwards compatibility. To the best of my knowledge, no code uses this.

I was hoping we could avoid unnecessary evils until it's really necessary. It's also confusing for client code. 😄